data/pigui/views/game.lua

```diff
@@ -309,6 +309,8 @@ ui.registerHandler('game', function(delta_t)
 			debugReload(gameView.modules)
 			debugReload(gameView.hudModules)
 			debugReload(gameView.sidebarModules)
+
+			Event.Queue("onDebugReload");
```

Comment:
Is this used somewhere? Note that our current model is meant to have a "selective" hot-reload where only the active module (for some definition of active module) is reloaded rather than "reloading the world".

For context: we're moving towards a model where individual files/modules are automatically hot-reloaded on change, and modules can opt-in to replace their copies of the module when notified of a submodule having been reloaded.

Reply:
Ah, it's a leftover from before I added the loop to call the various gauges to reload themselves. It was used in the ScanGauge as it was the only gauge (that I know of) that was "missing" after a reload.  I didn't even know it existed for ages since it'd always been killed by me reloading the scripts - I only found it when searching through the code and then tried to figure out why it wasn't being displayed.

Reply:
Removed in 61645c227c7814ce1634452cb69d0f545d97227a